pkg/maps/lbmap/lbmap.go

```diff
@@ -481,6 +491,15 @@ func (*LBBPFMap) DumpBackendMaps() ([]*loadbalancer.Backend, error) {
 	return lbBackends, nil
 }
 
+// IsMaglevLookupTableRecreated returns true if the maglev lookup BPF map
+// was recreated due to the changed M param.
+func (*LBBPFMap) IsMaglevLookupTableRecreated(ipv6 bool) bool {
```

Comment:
We should probably add a mechanism to detect if `IsMaglevLookupTableRecreated` is executed before `InitMaglevMaps`

Reply:
`maglev.InitMaglevMaps()` will be always called before the `pkg/service` handler has been started. If it is not the case, then inserting a service into a maglev lookup table should panic.

Reply:
Fair point